
test_error = [['P15YD', '用例失败,执行时异常'], ['064YD', 165, '下发任务失败'], ['VN24193YD', '用例执行通过'], ['密闭空间YD', '用例执行通过'], ['074YD', '用例失败,执行时异常']]

for i in test_error:
    print("error title:",i[0],"error message:",i[1])


